fastlane/Fastfile

```diff
@@ -128,6 +128,110 @@ lane :tag_current_branch do |options|
   push_git_tags(tag: version_number)
 end
 
+desc "Update hybrid common pod and gradle"
+lane :update_hybrid_common_versions do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  if options[:version]
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  previous_version_number = get_phc_version
+
+  # Add to options so if we're chaining lanes together, the other lanes get it
+  options[:previous_hybrid_version] = previous_version_number
+
+  UI.message("ℹ️  Current Purchases Hybrid Common version: #{previous_version_number}")
+  UI.message("ℹ️  Setting Purchases Hybrid Common version: #{new_version_number}")
+  files_to_update = [
+    'ios/purchases_flutter.podspec',
+    'macos/purchases_flutter.podspec',
+    'android/build.gradle',
+  ]
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do, dry_run: true")
+  else
+    replace_text_in_files(
+      previous_text: previous_version_number,
+      new_text: new_version_number,
+      paths_of_files_to_update: files_to_update
+    )
+  end
+end
+
+desc "Build hybrid example"
+lane :build_hybrid_example do |options|
```

Comment:
I'm guessing all these will update the `Podfile.lock` files right? I wonder if we should just substitute the version as we do in the `podspec` files... That would not work locally though, since we still would need to run this command, so I guess I'm ok with keeping this one.

Reply:
I think I prefer running the `pod` command in case any transitive dependency gets updated right?

Reply:
Ah yeah you're totally right 👍